packages/calcite-components/src/components/tip-manager/usage/Basic.md

```diff
@@ -2,44 +2,40 @@ Renders a tip manager using a group of tips as well as a single tip.
 
 ```html
 <calcite-tip-manager>
-  <calcite-tip-group group-title="Tip Manager heading">
-    <calcite-tip heading="Tip heading" selected>
-      <img slot="thumbnail" src="https://placeimg.com/200/200" alt="This is an image." />
+  <calcite-tip-group group-title="Animal Insights">
+    <calcite-tip heading="Paws for Thought" selected>
+      <img slot="thumbnail" src="https://placedog.net/400/200" alt="An image of a dog." />
```

Comment:
nitpick: Similar to [this comment](https://github.com/Esri/calcite-design-system/pull/8384#discussion_r1425923212), should the `alt` be skipped in this case? Maybe the `alt` context isn't need here (and in other examples below) and could be [marked as decorative](https://www.w3.org/WAI/tutorials/images/decorative/)?